distributed/pom.xml

```diff
@@ -135,6 +135,11 @@
             <artifactId>hazelcast</artifactId>
             <version>${hz.version}</version>
         </dependency>
+        <dependency>
```

Comment:
Could we have this as optional dependency ? 
Is not needed for everyone that do not run kubernates to have kubernates :)

Reply:
As I wrote in the PR text, I don't know a way. It needs to be in the image.